### DISCUSSION TITLE
Raspberry Pi OS Lite 64-bit support

### DISCUSSION THREAD

**Discussion by billz** [RaspAP/raspap-webgui#1176] [Polls]
The 64-bit (Bullseye) version of RPi OS came out of beta and was [released in February](https://www.raspberrypi.com/news/raspberry-pi-os-64-bit/) this year.

According to several forum posts [[1](https://forums.raspberrypi.com/viewtopic.php?p=1934698&hilit=64+bit+raspap#p1934698)] [[2](https://forums.raspberrypi.com/viewtopic.php?p=1728454&hilit=64+bit+raspap#p1728454)], RaspAP is reported to work without issues with this distro. However, we have not yet validated this in our own testing or offered official support at this stage. 

The purpose of this poll and thread is to gather feedback from users of RaspAP. We'd like to have a community-driven test phase in lead to working out any potential issues before adding RPi OS Lite 64-bit to our [list of supported distros](https://docs.raspap.com/#compatible-operating-systems).

Your input is greatly appreciated. Feel free to add anything in the comments below. Thanks :)

**Comment by msanaullahsahar** [comment]
I am using RaspAP on the following and it is working flawlessly.

Raspberry Pi OS Lite
Release date: April 4th 2022
System: 64-bit
Kernel version: 5.15
Debian version: 11 (bullseye)

**Reply by igorpecovnik** [reply]
FYI. Not long ago Armbian also added support for https://www.armbian.com/rpi4b/  64bit / ubuntu based but stock Rpi kernel, so things has to work / works the same and many boards are covered in similar fashion.

**Comment by Fredd-30** [comment]
I am using RaspAP on the following and it is working.

Raspberry Pi OS Lite
Release date: February 21st 2023
System: 64-bit
Kernel version: 5.15
Debian version: 11 (bullseye)

After update:
OS: Debian GNU/Linux 11 (bullseye) aarch64 
Host: Raspberry Pi 3 Model B Rev 1.2 
Kernel: 6.1.21-v8+ 

* To have no issue with the install need to install package git* before invoke insiders installer

**Reply by billz** [reply]
@Fredd-30 appreciate the feedback. 

With the help of our community, RaspAP was validated to work with the 64-bit kernel since May of last year. The recent firmware update bumped the kernel to the 6.1 tree. Additionally, the default kernel loaded was switched to 64-bit on Pi4 with the 32-bit install.

The 6.1.21-v8+ kernel [introduced a bug](https://github.com/RaspAP/raspap-webgui/issues/1341) in the Quick installer which has since been resolved. 

```
$ uname -a
Linux raspberrypi 6.1.21-v8+ #1642 SMP PREEMPT Mon Apr  3 17:24:16 BST 2023 aarch64 GNU/Linux
```

**Reply by Fredd-30** [reply]
Ok Bill I don't have read this before but solved.
Just howto have the latest version of webgui into insiders branch ? 

Fred

**Reply by billz** [reply]
The firmware update only affected the installer, which is called from this repo (ie., not Insiders). Upgrading your Insiders to the latest release is described here: https://docs.raspap.com/insiders/#upgrading

Adding an upgrade function to RaspAP's UI is on the project roadmap.
